keyboards/ares/matrix.c

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
This file is no longer needed, as atmega32a support has been added for the core matrix code.

You can and should use the default matrix, not a custom matrix, and this file should be removed